lib/main.dart

```diff
@@ -80,7 +83,7 @@ class HomeScreen extends StatelessWidget {
             const SizedBox(height: 24),
             Text(AppLocalizations.of(context)!.hello),
             Text(
-              FlutterConfig.get('SECRET'),
+              FlutterConfig.get('CLIENT_SECRET'),
```

Comment:
Can we use `env. authClientSecret` here, please 🙏

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/28/commits/55417b71c60b16af19eb96ca7bb0016f46388dc0.